Documentation/dev/release.md

```diff
@@ -70,6 +70,9 @@ for i in etcd-*{.zip,.tar.gz}; do gpg2 --default-key $SUBKEYID --output ${i}.asc
 for i in etcd-*{.zip,.tar.gz}; do gpg2 --verify ${i}.asc ${i}; done
 ```
 
+The public key used to sign etcd binaries can be found at [coreos.com/security](https://coreos.com/security/app-signing-key)
```

Comment:
Did clarify about which key we are using for signing etcd binaries. PTAL. Thanks.